src/public/styles/scss/style.scss

```diff
@@ -0,0 +1,24 @@
+/* Color Theme */
+$Color-1: #73203e;
+$Color-2: #f2508b;
+$Color-3: #f2ddd0;
+$Color-4: #f26444;
+$Color-5: #0d0d0d;
```

Comment:
- sempre que você for criar uma variável nunca coloque com maiúsculas.
- eu recomendo você usar o nome da propria cor mesmo Ex: `$blue01` caso tenha só um tipo de varição de cor basta colocar no nome da cor mesmo EX: `white`